sanic/sanic.py

```diff
@@ -247,7 +243,7 @@ def converted_response_type(self, response):
     def run(self, host="127.0.0.1", port=8000, debug=False, before_start=None,
             after_start=None, before_stop=None, after_stop=None, sock=None,
             workers=1, loop=None, protocol=HttpProtocol, backlog=100,
-            stop_event=None):
+            stop_event=None, logger=None):
```

Comment:
Maybe copy something similar to this:

https://github.com/pallets/werkzeug/blob/cbd8536b78b187766c67d21db3b81d284563a794/werkzeug/_internal.py#L75-L87